src/setup-binary.ts

```diff
@@ -0,0 +1,87 @@
+import * as core from "@actions/core";
+import * as hc from "@hashicorp/js-releases";
+import * as io from "@actions/io";
+import * as tc from "@actions/tool-cache";
+import * as sys from "./system";
+import cp from "child_process";
+import path from "path";
+import {ok} from "assert";
+
+const USER_AGENT = "setup-nomad-pack (GitHub Actions)";
```

Comment:
```suggestion
const USER_AGENT = "setup-${BINARY_NAME} (GitHub Actions)";
```

Not sure this is the right syntax but you get the idea

Reply:
Yup! Added and fixed